app/assets/javascripts/client/home/StartAssessmentCtrl.js

```diff
@@ -0,0 +1,97 @@
+(function() {
+  'use strict';
+  angular.module('PDRClient')
+      .controller('StartAssessmentCtrl', StartAssessmentCtrl);
+
+  StartAssessmentCtrl.$inject = [
+    '$scope',
+    '$timeout',
+    '$location',
+    '$modal',
+    'Assessment',
+    'SessionService'
+  ];
+
+  function StartAssessmentCtrl($scope, $timeout, $location, $modal, Assessment, SessionService) {
+    var vm = this;
+
+    vm.user = SessionService.getCurrentUser();
+    vm.district = vm.user.districts[0];
+
+    vm.alerts = [];
+    vm.assessment = {};
+
+    vm.hideModal = function() {
+      $('#startAssessment').modal('hide');
```

Comment:
This is existing code but the catch here is that this is using the Bootstrap CSS approach to modals instead of the `$modal` service, making it a lot harder to test.  A better rewrite of this would involve using the actual `$modal` service, which would make this more straightforward to test.